Src/Couchbase.Linq/Couchbase.Linq.nuspec

```diff
@@ -15,11 +15,14 @@
     <tags>couchbase linq n1ql nosql database</tags>
     <dependencies>
       <group targetFramework="net45">
+        <dependency id="CouchbaseNetClient" version="2.4.5" />
         <dependency id="Newtonsoft.Json" version="9.0.1" />
         <dependency id="Common.Logging" version="3.3.1" />
+        <dependency id="Castle.Core" version="3.3.3" />
+        <dependency id="Remotion.Linq" version="2.1.1" />
       </group>
       <group targetFramework="netstandard1.5">
-        <dependency id="Castle.Core" version="4.0.0" />
+        <dependency id="Castle.Core" version="3.3.3" />
```

Comment:
@jeffrymorris 

The minimum version for the .NET Standard 1.5 version really is 4.0.0 because that's the first version of Castle.Core that supports .NET Standard.  Version 1.3.1 now won't install on .NET Core applications.  I'll file a new pull to fix this back.

Brant